app/controllers/content_items_controller.rb

```diff
@@ -56,4 +63,16 @@ def error_403(exception)
   def error_notfound
     render plain: 'Not found', status: :not_found
   end
+
+  def ab_testing_group
```

Comment:
Could the logic for generic A/B testing be collected in a separate file?
Can we have a further separate place for the logic specific to each test?